static/app/views/settings/project/projectKeys/details/loaderSettings.tsx

```diff
@@ -0,0 +1,266 @@
+import {Fragment, useCallback, useEffect, useState} from 'react';
```

Comment:
shall we have a unit test for this file? maybe we already have tests, so we could just move them to a new file

Reply:
I split out the loader settings tests into a dedicated file!